roles/oVirt.storages/tasks/main.yml

```diff
@@ -16,6 +16,7 @@
     posixfs: "{{ item.value.posixfs | default(omit) }}"
     glusterfs: "{{ item.value.glusterfs | default(omit) }}"
     fcp: "{{ item.value.fcp | default(omit) }}"
+    discard_after_delete: "{{ item.value.discard_after_delete | default(omit) }}"
```

Comment:
should be default(true) ?

Reply:
It's by default true in module side, so I would omit it here if it's not specified in our config and let module to do the default.

Reply:
Changed to undef after offline discussion.